dropwizard-index-page/src/main/java/com/palantir/indexpage/IndexPageServlet.java

```diff
@@ -54,11 +54,11 @@ public IndexPageServlet(String contextPath, String indexPagePath) {
         }
 
         ImmutableMap<String, String> templateContext = ImmutableMap.of(BASE_URL, slashedContextPath);
-        Optional<URL> indexPageResource = tryGetResource(indexPagePath);
-        if (indexPageResource.isPresent()) {
-            indexPage = new ClasspathIndexPage(templateContext, indexPageResource.get());
+        File indexPageFile = new File(indexPagePath);
```

Comment:
oh. just re-read title of PR :)

if this PR has multiple PRs, might be worth doing the errorprone change in here as well